web/packages/teleterm/src/services/vnet/__snapshots__/diag.test.ts.snap

```diff
@@ -0,0 +1,37 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`reportToText converts report correctly 1`] = `
+"VNet Diagnostic Report
+
+Created at: 2024-11-23 13:27:48
+Network interface: utun4
+IPv4 CIDR ranges: 100.64.0.0/10
+IPv6 prefix: fdff:fd74:46c0::
+DNS zones: teleport.example.com, company.test
+
+---
+⚠️ There are network routes in conflict with VNet.
+
+| Conflicting destination | VNet destination | Interface | Set up by |
+| ----------------------- | ---------------- | --------- | --------- |
+| 100.64.0.0/10 | 100.64.0.1 | utun5 | VPN: Foobar |
+| 0.0.0.0/1 | 100.64.0.0/10 | utun6 | unknown |
```

Comment:
I don't really want to spend time on making the whole table line up perfectly as that's not the point. I sparingly used Markdown in the report so that the output is rendered correctly when pasted into Zendesk and Slack. Slack doesn't support tables, but Zendesk does.

<img width="1384" alt="vnet-text-report" src="https://github.com/user-attachments/assets/9a2776b0-be93-41cc-adba-3a389d43b7b9" />

CLI command output looks particularly bad without code blocks, as in that case most Markdown parsers (or I guess all of them?) do not interpret a single newline character as the start of a new line. So the output from `netstat` ends up all garbled if code blocks are not used.

Reply:
Hmm, I should probably document that it's supposed to be Zendesk/Slack friendly.